docs/pages/debugging/create-devtools-plugins.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Create a dev tools plugin
+description: Learn how to create a dev tools plugin to enhance your development experience.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+> **warning** Dev Tools plugins are compatible with SDK 50 and above. The process for creating a plugin is still in beta and may change in the future.
+
+You create a dev tools plugin, whether that's for inspecting aspects of a common framework or library or something specific to your custom code.
+
+A plugin consists of three key elements:
+- an Expo app to display the dev tools web user interface
+- an **expo-module.config.json** for Expo CLI recognition
+- calls to `expo/devtools` API for the app to communicate back and forth with the dev tool's web interface
```

Comment:
```suggestion
- an Expo app to display the dev tools web user interface,
- an **expo-module.config.json** for Expo CLI recognition,
- calls to `expo/devtools` API for the app to communicate back and forth with the dev tool's web interface.
```

Formatting nit. Alternatively, we can make the points a full statements (start them with upper case), and then we should end the lines with period.